website/cue/reference/components/sources/kubernetes_logs.cue

```diff
@@ -263,6 +263,19 @@ components: sources: kubernetes_logs: {
 				unit:    "milliseconds"
 			}
 		}
+		delay_deletion_ms: {
+			common:      false
+			description: """
+				Delay between receiving a `DELETE` event and removing any related metadata Vector has stored. This controls how quickly Vector will remove
+				metadata for resources that have been removed from Kubernetes, a longer delay will allow Vector to continue processing and enriching logs after the source Pod has been deleted.
+				If Vector tries to process logs from a Pod which has already had it's metadata removed from the local cache, it will fail to enrich the event with metadata and log a warning.
```

Comment:
```suggestion
				If Vector tries to process logs from a Pod which has already had its metadata removed from the local cache, it will fail to enrich the event with metadata and log a warning.
```

Reply:
:shakes_fist: